src/routes/DigitalAssets/sagas/digitalAssets.js

```diff
@@ -42,11 +34,32 @@ function* initDigitalAssetsSaga(): Saga<void> {
       ...allAssets,
     }
 
-    yield put(setInitialItems(allAssetsWithETH))
+    yield put(digitalAssets.setInitialItems(allAssetsWithETH))
   }
 }
 
+function* setAssetIsActive(): Saga<void> {
+  yield put(blocks.syncRestart())
```

Comment:
It looks, that syncRestart can be executed before storage updates and get old asset list.
We should check it before

Reply:
@flexxnn I have checked.
next action is called once previous was finished